.github/workflows/zenodo-cache-sync.yml

```diff
@@ -7,7 +7,7 @@ on:
     - cron: "0 6 * * 1-5" # Weekdays at midnight on MST
   pull_request:
     paths:
-      - "src/pudl/workspace/datastore.py"
+      - "src/pudl/package_data/settings/zenodo_dois.yml"
```

Comment:
Now instead of this workflow running any time anything in the datastore module changes, it'll just refresh the cache when we actually change the DOIs (or add/remove datasets).